pkg/plugin/validator.go

```diff
@@ -47,7 +47,7 @@ type ExchangeResponse struct {
 
 // pluginGithubIssue contains the required attribute parsed from
 // the issue URL.
-type pluginGithubIssue struct {
```

Comment:
While we're refactoring - this currently stutters because the package name is also plugin (`plugin.pluginGitHubIssue`). Should this just be `GitHubIssue`?

Reply:
I think this should not be a concern at this is not exported, so there won't be a case when we need to call `plugin.pluginGithubIssue`? 
I thought about naming it as `githubIssue`, but it seems a bit confusing as w3 are also using `github.Issue` from `go-plugin`.